adapters/Vungle/VungleAdapter/GADMAdapterVungleInterstitial.m

```diff
@@ -168,6 +168,10 @@ - (void)adNotAvailable:(nonnull NSError *)error {
 }
 
 - (void)willShowAd {
+  // Do nothing.
+}
+
+- (void)didViewAd {
   [_connector adapterWillPresentInterstitial:self];
```

Comment:
Doesn't  it make more sense that `willShowAd` will invoke `willPresentInterstitial` ? shouldn't this go in willShowAd?

Reply:
@chlandacastro Hi chlandacastro, based on our engineer @ashinagawa had the conversation with AdMob side(you might not be in that conversation) contacts before, the following was identified :
For Interstitial Ads:
The new SDK callback `vungleAdViewedForPlacement:` should be mapped to the `adapterWillPresentInterstitial:` adapter callback (which would also need to be removed from the `willShowAd` callback).
Thanks a lot. O(∩_∩)O

Reply:
Hey @YueVungle , was that conversation with me? If so, I don't remember it.

`adapterWillPresentInterstitial` is a callback to tell the user "an ad is about to take over the full screen". The Vungle router maps `vungleWillShowAdForPlacementID` to `willShowAd` and `vungleAdViewedForPlacement` to `didViewAd`. The meaning of Google's `adapterWillPresentInterstitial` callback sounds much closer to `vungleWillShowAdForPlacementID` like how it was before.

Reply:
Hi @ericleich and @chlandacastro, `vungleAdViewedForPlacement:` is more for “impression timing (= at the very first frame is rendered)“, so we linked it to `adapterWillPresentInterstitial:`. 

My leader is taking PTO  and will come back to work tomorrow, after he comes back to work, I will confirm with him who was in that conversation, thanks a lot and have a great day! ^_^